internal/kafka/test/integration/kafka_create_test.go

```diff
@@ -642,6 +642,7 @@ func TestKafkaCreate_EnterpriseKafkas(t *testing.T) {
 		cluster.ProviderSpec = api.JSON{}
 		cluster.ClusterSpec = api.JSON{}
 		cluster.OrganizationID = organizationID
+		cluster.ClusterType = api.EnterpriseDataPlaneClusterType.String()
```

Comment:
> the region and cloud provider doesn't matter for enterprise? is it because the cluster id is sent in the request?

No, they do not matter. Only the cluster id info is used for cluster placement. Consequently the kafka cloud provider/region info will be inherited from the cluster's information.

Reply:
I think we'll need to update the logs and error message as the following doesn't make sense in this case  🤔 

`no available cluster found for Kafka instance type 'standard' in region 'us-east-1':instance type "standard" not supported on selected cluster "1234abcd1234abcd1234abcd1234abcd"`

`reason: "cluster \"us-east-1\" cannot accept instance type: \"standard\" at this moment"`

Reply:
> reason: "cluster \"us-east-1\" cannot accept instance type: \"standard\" at this moment"

Ah this one is interesting. It should be the cluster id and not the region in this message.

Reply:
Pushed a new commit to update the reason.